python/context/ball_trajectories.py

```diff
@@ -427,12 +447,12 @@ def get_trajectory(self, index: int) -> StampedTrajectory:
         """
         return self._data[index]
 
-    def random_trajectory(self) -> StampedTrajectory:
+    def random_trajectory(self) -> typing.StampedTrajectory:
```

Comment:
Isn't `StampedTrajectory` defined in this file and not in `typing`?

Reply:
+1